frontend/views/components/Graphs/PieChart.vue

```diff
@@ -166,4 +150,18 @@ export default {
   text-align: center;
   line-height: 1.2;
 }
+
+// Global Classes for SVG fills
```

Comment:
Why did you move it from utilities? I could use the same class for my graph actually

Reply:
You are right! Moved back to utilities and with better names.